src/app/system/manage-reports/create-report/create-report.component.scss

```
@@ -5,3 +5,58 @@
+  margin: 0.5rem 0;
+  font-size: 0.875rem;
+}
+
+.ai-sql-preview {
+  margin: 0;
+  padding: 0.75rem;
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 0.85rem;
+  line-height: 1.6;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)